### ISSUE TITLE
Poor documentation on how to submit bugs...

### ISSUE THREAD

**Issue by toddmushenoatflairsoft** [bndtools/bnd#1870]
As an end user of your product, I found that I had to go through several steps that were undocumented to submit a bug.
Please fix your documentation to detail HOW one should go about submitting a bug (someone who is unfamiliar of your process).

Todd

**Comment by bjhargrave** [comment]
The README has a [feedback](https://github.com/bndtools/bnd/blob/master/README.md#feedback) section which says:

`Bugs and issues should go to https://github.com/bndtools/bnd/issues`

which seems fairly straightforward and is fairly standard for GitHub projects. 

How would you suggest rewording this? You can submit a PR with suggested wording changes.

**Comment by toddmushenoatflairsoft** [comment]
I agree... and I am suggesting that that link would be very helpful on the osgi website.

Todd

**Comment by bjhargrave** [comment]
> I agree... and I am suggesting that that link would be very helpful on the osgi website.

What OSGi website/page do you mean?

**Comment by toddmushenoatflairsoft** [comment]
https://www.osgi.org/ From there it is not obvious how to submit bugs...
or put another way...
When there are issues with http://enroute.osgi.org/qs/050-start.html how does one resolve it.

**Comment by bjhargrave** [comment]
On www.osgi.org, you can find the OSGi Public Bugzilla under Developer->Specifications->Bugs and Issues. However this is for reporting issues with the OSGi specifications. 

On enroute.osgi.org, under Forum, it says that enRoute related issues are to be discussed on the osgi-dev mail list.

**Comment by toddmushenoatflairsoft** [comment]
Agreed, I guess it was mainly the enroute site.
